ArjunaJTS/jts/tests/classes/com/hp/mwtests/ts/jts/orbspecific/resources/AtomicWorker3.java

```diff
@@ -74,20 +76,20 @@ public static void randomOperation (int thr, int level)
 		{
 		    current.begin();
 
-		    Util.indent(thr, level);
-		    System.out.println("begin");
+		    logger.info(""+level);
```

Comment:
Reading the word level rather then looking at an actual text indentation is not so visually appealing. But if you really think we should be avoiding System.out then that's fine (and you probably need to remove the Util.indent methods to make sure they aren't used anywhere else).